src/components/inline_edit/inline_edit_form.test.tsx

```diff
@@ -448,6 +480,26 @@ describe('EuiInlineEditForm', () => {
         expect(getByTestSubject('euiInlineReadModeButton')).toBeTruthy();
         expect(getByText('New message!')).toBeTruthy();
       });
+
+      it('correctly applies `inputModeProps.placeholder`', () => {
```

Comment:
Great job with these tests! Minor test name suggestion

```suggestion
      it('allows overriding `placeholder` with `inputModeProps.placeholder`', () => {
```